tests/framework/extensions/namespaces/namespaces.go

```diff
@@ -44,3 +44,22 @@ func GetNamespaceByName(client *rancher.Client, clusterID, namespaceName string)
 
 	return namespace, nil
 }
+
+// DeleteNamespace is a helper function to delete  a namespaces from a cluster from a given project.
+func DeleteNamespace(client *rancher.Client, namespace string, clusterID string) ( error) {
```

Comment:
The last PR about this [#38171](https://github.com/rancher/rancher/pull/38171). Seems like there's not an implemented delete function, however, namespace creation function already has its session cleanup [here](https://github.com/rancher/rancher/blob/20a4ab0b4e3d19ce23b94227985f03c0ca05ecaf/tests/framework/extensions/namespaces/create.go#L92-L117). I am unsure if overwriting and requesting delete call twice would cause a problem for cleanup enabled test runs.

Reply:
That's fine, Ugur, the cleanup would just log an error or not, but the delete needs to be explicitly tested, so never mind, thanks for answering.